app/src/main/java/mozilla/lockbox/view/RootActivity.kt

```diff
@@ -51,4 +57,18 @@ class RootActivity : AppCompatActivity() {
             super.onBackPressed()
         }
     }
+
+    fun showToastNotification(@StringRes strId: Int? = null, text: String? = null, viewGroup: ViewGroup) {
+        assertOnUiThread()
```

Comment:
* You should get the `ViewGroup` from the `activity_root.xml`. You can see [an example of this here][1].
 * You could feasibly move all of this into the `RoutePresenter`, because you don't need it to be callable from the fragments (again, look how we do it for `DialogAction`s.

[1]: android.widget.Toast